src/main/java/somethingrandom/usecase/AddDelayedItemInputBoundary.java

```diff
@@ -0,0 +1,5 @@
+package somethingrandom.usecase;
+
+public interface AddDelayedItemInputBoundary {
+    void execute(AddDelayedItemInputData inputData) throws DataAccessException;
```

Comment:
Ideally, we'd have some other exception, or decide what to do with it in the interactor, but this is probably fine for now.